store/src/main/java/com/nytimes/android/external/store/base/impl/RealInternalStore.java

```diff
@@ -186,15 +189,15 @@ public void call(Parsed parsed) {
     }
 
     void backfillCache(@Nonnull Key key) {
-        fetch(key).subscribe(new Action1<Parsed>() {
+        Disposable disposable = fetch(key).subscribe(new Consumer<Parsed>() {
```

Comment:
not sure if it makes sense to make a disposable if nothing is going to dispose it